src/botPage/common/const.js

```diff
@@ -215,6 +216,7 @@ const config = {
     },
     conditionsCategoryName: {
         callput     : translate('Up/Down'),
+        callputequal: translate('Up/Down Equals'),
```

Comment:
@aaron-binary  why don't we get these names from API?

Reply:
We should, as discussed we'll address this in different card.